fedimint-server/src/config/api.rs

```diff
@@ -63,20 +63,26 @@ impl ConfigGenApi {
         config_generated_tx: Sender<ServerConfig>,
         task_group: &mut TaskGroup,
     ) -> Self {
-        Self {
+        let config_gen_api = Self {
             data_dir,
             state: Arc::new(Mutex::new(ConfigGenState::new(settings))),
             db,
             config_generated_tx,
             task_group: task_group.clone(),
-        }
+        };
+        tracing::info!(target: fedimint_logging::LOG_NET_PEER_DKG, "Created new config gen Api");
+        config_gen_api
     }
 
     // Sets the auth and decryption key derived from the password
     pub fn set_password(&self, auth: ApiAuth) -> ApiResult<()> {
         let mut state = self.require_status(ServerStatus::AwaitingPassword)?;
         state.auth = Some(auth);
         state.status = ServerStatus::SharingConfigGenParams;
+        tracing::info!(
```

Comment:
Fully qualifying logging statements seems unnecessary and verbose. We just do `info!` etc. everywhere.